web/org.eclipse.xtext.web.example.statemachine/src/org/eclipse/xtext/web/example/statemachine/GenerateStatemachine.mwe2

```diff
@@ -84,6 +84,8 @@ Workflow {
 				fragment = parser.antlr.XtextAntlrUiGeneratorFragment auto-inject {}
 			}
 			
+			fragment = parser.antlr.XtextAntlrGeneratorComparisonFragment auto-inject {}
```

Comment:
Those workflows still work completely in the old generator infrastructure, don't they?
So it wouldn't make sense to integrate the comparison now but once they're migrated to the new infrastructure.